src/MSBuild/XMake.cs

```diff
@@ -1960,6 +1960,11 @@ private static bool IsEnvironmentVariable(string envVar)
         /// </summary>
         internal static bool usingSwitchesFromAutoResponseFile = false;
 
+        /// <summary>
+        /// Indicates that this process is working as a server.
+        /// </summary>
+        private static bool s_isServerNodeHosted;
```

Comment:
NIT: the hosted suffix is confusing to me. Please consider to rename it to `s_isServerNode`

Reply:
I wasn't sure if the only node inside process will be the server node. But yes, I will change it.